docs/src/explanations/radial-integral.rst

```diff
@@ -0,0 +1,278 @@
+   \end{aligned}
+
+where :math:`P_l(x)` is the :math:`l`-th Legendre polynomial.
+
+Derivation of the explicit radial integral for Gaussian densities
+-----------------------------------------------------------------
+
+Denoting by :math:`\theta(\boldsymbol{r},\boldsymbol{r}_{ij})` the angle between a
+generic position vector :math:`\boldsymbol{r}` and the vector
+:math:`\boldsymbol{r}_{ij}`, we can write
+
+.. math::
+
+   \begin{aligned}
+      g(\boldsymbol{r}- \boldsymbol{r}_{ij}) & = \frac{1}{(\pi
+      \sigma^2)^{3/4}}e^{-\frac{(\boldsymbol{r}- \boldsymbol{r}_{ij})^2}{2\sigma^2}} \\
+      & = \frac{1}{(\pi
+      \sigma^2)^{3/4}}e^{-\frac{(r_{ij})^2}{2\sigma^2}}e^{-\frac{(\boldsymbol{r}^2-
+      2\boldsymbol{r}\boldsymbol{r}_{ij})}{2\sigma^2}},
+   \end{aligned}
+
+where the first factor no longer depends on the integration variable :math:`r`.
+
+Analytical Expressions for the GTO Basis
+----------------------------------------
+
+While the above integrals are hard to compute in general, the GTO basis is one of the
+few sets of basis functions for which many of the integrals can be evaluated
+analytically. This is also useful to test the correctness of more numerical
+implementations.
+
+The primitive basis functions are defined as
+
+.. math::
+
+   \begin{aligned}
+       R_{nl}(r) = R_n(r) = r^n e^{-\frac{r^2}{2\sigma_n^2}}
+   \end{aligned}
+
+In this form, the basis functions are not yet orthonormal, which requires an extra
+linear transformation. Since this transformation can also be applied after computing the
+integrals, we simply evaluate the radial integral with respect to these primitive basis
+functions.
+
+Real Space Integral for Gaussian Densities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We now evaluate
+
+.. math::
+
+   \begin{aligned}
+       I_{nl}(r_{ij}) & = \frac{1}{(\pi \sigma^2)^{3/4}}4\pi
+       e^{-\frac{r_{ij}^2}{2\sigma^2}} \int_0^\infty \mathrm{d}r r^2 R_{nl}(r)
+       e^{-\frac{r^2}{2\sigma^2}} i_l\left(\frac{rr_{ij}}{\sigma^2}\right) \\ & =
+       \frac{1}{(\pi \sigma^2)^{3/4}}4\pi e^{-\frac{r_{ij}^2}{2\sigma^2}} \int_0^\infty
+       \mathrm{d}r r^2 r^n e^{-\frac{r^2}{2\sigma_n^2}} e^{-\frac{r^2}{2\sigma^2}}
+       i_l\left(\frac{rr_{ij}}{\sigma^2}\right),
+   \end{aligned}
+
+the result of which can be conveniently expressed using :math:`a=\frac{1}{2\sigma^2}`,
+:math:`b_n = \frac{1}{2\sigma_n^2}`, :math:`n_\mathrm{eff}=\frac{n+l+3}{2}` and
+:math:`l_\mathrm{eff}=l+\frac{3}{2}` as
+
+.. math::
+
+   \begin{aligned}
+       I_{nl}(r_{ij}) = \frac{1}{(\pi \sigma^2)^{3/4}} \cdot
+       \pi^{\frac{3}{2}}\frac{\Gamma\left(n_\mathrm{eff}\right)}{\Gamma\left(l_\mathrm{eff}\right)}\frac{(ar_{ij})^l}{(a+b)^{n_\mathrm{eff}}}M\left(n_\mathrm{eff},l_\mathrm{eff},\frac{a^2r_{ij}^2}{a^2+b^2}\right),
+   \end{aligned}
+
+where :math:`M(a,b,z)` is the confluent hypergeometric function (hyp1f1).
+
+.. _k-space-radial-integral-1:
+
+K-space Radial Integral
+-----------------------
```

Comment:
Is this missing?

Reply:
Yes, we have to write it 🤣 pinging @kvhuguenin here because heis the master for this. We can remove the section for now.